packages/app/src/app/components/CreateNewSandbox/CreateSandbox/CreateSandbox.tsx

```diff
@@ -95,11 +95,38 @@ export const CreateSandbox: React.FC<CreateSandboxProps> = ({
             </DashboardButton>
           )}
           <Tabs {...tab} aria-label="Create new">
-            <Tab {...tab} stopId="create">
-              Create Sandbox
+            <Tab {...tab} stopId="quickstart">
+              Quickstart
             </Tab>
             <Tab {...tab} stopId="import">
-              Import Project
+              Import from GitHub
+            </Tab>
+            <Tab {...tab} stopId="my-templates">
+              My templates
+            </Tab>
+            <Tab {...tab} stopId="csb-templates">
+              CodeSandbox templates
+            </Tab>
+            <Tab {...tab} stopId="react-essentials">
+              React essentials
+            </Tab>
+            <Tab {...tab} stopId="vue-essentials">
+              Vue essentials
+            </Tab>
+            <Tab {...tab} stopId="angular-essentials">
+              Angular essentials
+            </Tab>
+            <Tab {...tab} stopId="ui-frameworks">
+              UI frameworks
+            </Tab>
+            <Tab {...tab} stopId="component-libraries">
+              Component libraries
+            </Tab>
+            <Tab {...tab} stopId="starters">
+              Web App and API Starters
+            </Tab>
+            <Tab {...tab} stopId="databases">
+              Databases
```

Comment:
It's super weird that it shows these as changes because [they're changes in the `dashboard/improvements` branch already](https://github.com/codesandbox/codesandbox-client/blob/dashboard/improvements/packages/app/src/app/components/CreateNewSandbox/CreateSandbox/CreateSandbox.tsx#L104-L130)...